python/astra/optomo.py

```diff
@@ -111,43 +111,15 @@ def _matvec(self,v):
         :param v: Volume to forward project.
         :type v: :class:`numpy.ndarray`
         """
-        v = self.__checkArray(v, self.vshape)
-        vid = self.data_mod.link('-vol',self.vg,v)
-        s = np.zeros(self.sshape,dtype=np.float32)
-        sid = self.data_mod.link('-sino',self.pg,s)
-
-        cfg = creators.astra_dict('FP'+self.appendString)
-        cfg['ProjectionDataId'] = sid
-        cfg['VolumeDataId'] = vid
-        cfg['ProjectorId'] = self.proj_id
-        fp_id = algorithm.create(cfg)
-        algorithm.run(fp_id)
-
-        algorithm.delete(fp_id)
-        self.data_mod.delete([vid,sid])
-        return s.flatten()
+        return self.FP(v, out=None).flatten()
```

Comment:
Well ravel is perhaps somewhat more obvious, it also has its own compiled code so its marginally faster:

```python
In [2]: A = np.zeros([10, 10, 10, 10])
In [3]: %timeit A.ravel()
1000000 loops, best of 3: 206 ns per loop
In [4]: %timeit A.reshape(-1)
1000000 loops, best of 3: 771 ns per loop
```